src/server.ts

```diff
@@ -181,10 +184,24 @@ const fetchVulnerabilities = async (reqData) => {
     const headers = {
         'Content-Type': 'application/json',
         'Authorization': 'Bearer ' + config.api_token,
+        'request_id': uuid.v4(),

```

Comment:
You are generating one request id per batch, actually you should do one per batch.

Reply:
Changed.